synapse/lib/parser.py

```diff
@@ -395,6 +395,7 @@ def massage_vartokn(x):
     'DOUBLEQUOTEDSTRING': lambda x: s_ast.Const(unescape(x)),  # drop quotes and handle escape characters
     'NUMBER': lambda x: s_ast.Const(s_ast.parseNumber(x)),
     'HEXNUMBER': lambda x: s_ast.Const(s_ast.parseNumber(x)),
+    'BOOL': lambda x: s_ast.Bool(x == 'true'),
```

Comment:
this is weird

Reply:
The const value of a bool is limited to the string "true" and the string "false" so this works perfectly :)